frontend/lib/resolution/prims.cpp

```diff
@@ -614,6 +614,24 @@ static QualifiedType primObjectToInt(Context* context, const CallInfo& ci) {
   return QualifiedType(argType.kind(), IntType::get(context, 64));
 }
 
+/*
+  for get real/imag primitives
+*/
+static QualifiedType primComplexGetReal(Context* context, const CallInfo& ci) {
```

Comment:
Nitpick, but the `GetReal` in the name was confusing to me since this is the same whether we are getting the real or imaginary part of the complex number. I think `GetField` or `GetComponent` or even `GetRealOrImag` would be clearer.

Reply:
oh I see, I based the name on the type of returned element - not the component in the complex number